develop/tutorials/articles/100-tooling/03-blade-cli/06-managing-your-liferay-server-with-blade-cli.markdown

```diff
@@ -17,16 +17,22 @@ The first thing that comes to mind when interacting with a server is simply
 turning it on/off. You can use the `server` sub-command to accomplish this. To
 turn on a Liferay server (Tomcat or Wildfly/JBoss), you can run
 
-    blade server start -b
+    blade server start
 
-Likewise, to turn off a server, run
+This starts the server in the background. You can tail the logs by adding the
+`-t` flag. If you prefer starting the server in the foreground, run `blade
+server run`. Additionally, if you prefer starting the server in debug mode, add
+the `-d` flag.
 
-    blade server stop
+Debug mode can be customized by adding the `-p` tag to set the custom remote
+debugging port (defaults are `8080` for Tomcat and `8787` for Wildfly) and/or
```

Comment:
Hey @codyhoag can you verify this? Port 8080 is already used for HTTP; therefore it can't also be used for remote debugging.

Reply:
@sez11a it should be `8000` for Tomcat instead. Should I make the update?